src/components/[guild]/claim-poap/components/ClaimPoap.tsx

```diff
@@ -47,6 +49,8 @@ const ClaimPoap = ({ rolePlatformId }: Props) => {
         <ClaimPoapButton
           rolePlatformId={rolePlatformId}
           isDisabled={!isButtonEnabled}
+          isClaimed={claimed}
```

Comment:
I think we could move this logic inside `ClaimPoapButton` instead of using props